### PR TITLE
Update node packages & Run the api in docker 

### PR DESCRIPTION
**Pull request by safaa-alnabulsi**
Hi, I updated the packages and added a small section in the readme on how to run the api locally.
Update:
@Azgaar I added a docker file to run the api locally. By so, all needed packages will be working and no need to install anything on your local machine! 
I also attached a readme on how to use it ;)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,6 +149,15 @@ Claim or reclaim name `test2` with password `secret123`:
 Unclaim name `test2` using password `secret123`:
 > [armoria.herokuapp.com/unclaim/test2/secret123](https://armoria.herokuapp.com/unclaim/test2/secret123)
 
+## Local devlopment
```
Comment (Azgaar) [inline README.md]: Typo: dev**e**lopment

File: README.md [hunk 2]
```diff
@@ -149,6 +149,15 @@ Claim or reclaim name `test2` with password `secret123`:
...
+
+Make sure you have the following packages installed
+
+```bash
+brew install pkg-config cairo pango libpng jpeg giflib
```
Comment (Azgaar) [inline README.md:157]: Are you sure these packages are required?

**Review by Azgaar — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (safaa-alnabulsi) [reply README.md:157]: I needed them in order to install the packages. I tried many suggestions online and this what helped. I was getting errors related to `node-pre-gyp` and `canvas`.
I found this suggestion in some question. I see they are mentioned in here https://github.com/Automattic/node-canvas/blob/master/Readme.md

File: README.md (hunk 1 above)
Comment (safaa-alnabulsi) [reply README.md]: @Azgaar could you please add it as a suggestion ?

File: README.md (hunk 2 above)
Comment (Azgaar) [reply README.md:157]: Need to cross-check. If they are required, they should be mentioned in `package.json` dependencies

File: README.md [hunk 3]
```diff
@@ -149,6 +149,40 @@ Claim or reclaim name `test2` with password `secret123`:
 Unclaim name `test2` using password `secret123`:
 > [armoria.herokuapp.com/unclaim/test2/secret123](https://armoria.herokuapp.com/unclaim/test2/secret123)
 
+## Local devlopment
```
Comment (safaa-alnabulsi) [inline README.md]:
```suggestion
## Local development
```

File: README.md (hunk 2 above)
Comment (safaa-alnabulsi) [reply README.md:157]: Sure

**Pull request closed without merging**